src/route/router.ts

```diff
@@ -475,6 +477,62 @@ router.post('/video/pending/approved', approvePendingReviewVideoController) // 
 // https://localhost:9999/video/pending/approved
 // cookie: uid, token
 
+// ========== 新的审核接口（需要 developer 角色，使用 uuid 和 token） ==========
+
+router.get('/review/video/pending', getPendingReviewVideoListController) // 获取待审核视频列表（需要 developer 角色）
+// https://localhost:9999/review/video/pending?num=20&offset=0
+// cookie: uuid, token
+
+router.get('/review/comment/pending', getPendingReviewCommentListController) // 获取待审核评论列表（需要 developer 角色）
+// https://localhost:9999/review/comment/pending?num=20&offset=0
+// cookie: uuid, token
+
+router.get('/review/danmaku/pending', getPendingReviewDanmakuListController) // 获取待审核弹幕列表（需要 developer 角色）
+// https://localhost:9999/review/danmaku/pending?num=20&offset=0
+// cookie: uuid, token
+
+router.post('/review/video/approve', approveVideoReviewController) // 通过视频审核（需要 developer 角色）
+// https://localhost:9999/review/video/approve
+// cookie: uuid, token
+// {
+// 	"videoId": 123
+// }
+
+router.post('/review/video/reject', rejectVideoReviewController) // 退回视频审核（需要 developer 角色）
+// https://localhost:9999/review/video/reject
+// cookie: uuid, token
+// {
+// 	"videoId": 123
+// }
+
+router.post('/review/comment/approve', approveCommentReviewController) // 通过评论审核（需要 developer 角色）
+// https://localhost:9999/review/comment/approve
+// cookie: uuid, token
+// {
+// 	"commentRoute": "1.2.3"
+// }
+
+router.post('/review/comment/reject', rejectCommentReviewController) // 退回评论审核（需要 developer 角色）
+// https://localhost:9999/review/comment/reject
+// cookie: uuid, token
+// {
+// 	"commentRoute": "1.2.3"
+// }
+
+router.post('/review/danmaku/approve', approveDanmakuReviewController) // 通过弹幕审核（需要 developer 角色）
```

Comment:
```suggestion
router.post('/review/danmaku/approve', approveDanmakuReviewController) // 通过弹幕审核
```
通过 RBAC 限制访问，不用标明接口权限了